### PR TITLE
Added astro-nanostores example

### PR DESCRIPTION
**Pull request by agustinmulet**
## Changes

Adds a Proof of Concept of a shared store between components of different frameworks/libs.

## Testing

Doesn't have tests since it's a simple PoC.

## Docs

No docs modified, just added an example.

### CODE REVIEW

**Review by duncanhealy — COMMENTED** [review]
Could you remove the yarn.lock file as we have not put package lock files for the other examples

**Review by duncanhealy — COMMENTED** [review]
suggestion for package name change

File: examples/astro-nanostores/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@astrojs/starter-kit",
```
Comment (duncanhealy) [inline examples/astro-nanostores/package.json:2]:
```suggestion
  "name": "@astrojs/nanostore-example",
```

Comment (natemoo-re) [reply examples/astro-nanostores/package.json:2]: Can we do `with-nanostore`?

Comment (agustinmulet) [reply examples/astro-nanostores/package.json:2]: Sure!

**Comment by agustinmulet** [comment]
I'll close this since I had some problems with my fork and created a new one, sorry. I'll apply the suggestions made here, thanks @duncanhealy and @natemoo-re !

**Pull request closed without merging**